src/views/Predictions/components/RoundCard/SetPositionCard.tsx

```diff
@@ -36,7 +36,7 @@ interface SetPositionCardProps {
   togglePosition: () => void
   epoch: number
   onBack: () => void
-  onSuccess: (decimalValue: string, hash: string) => Promise<void>
```

Comment:
This `decimalValue` wasn't used in the corresponding handleSuccess function (https://github.com/pancakeswap/pancake-frontend/pull/1977/files#diff-ef08349844ddbde8ad03f79f8d45dc7f46a0beafb9f9661033ac3594469298fdL100), so I've removed it.